shared/src/components/activation/Activation.scss

```
@@ -0,0 +1,127 @@
+.activation-status-dropdown-button {
+    &__progress-bar-container {
+        width: 1.5rem;
+        margin-left: 0.5rem;
+        margin-right: 0.3rem;
+    }
+    &__progress-bar-container.hidden {
+        visibility: hidden;
+    }
+}
+
+.activation-container {
+    &__header {
+        padding: 0.5rem;
+    }
+}
+
+.activation-checklist-container {
+    &__checklist {
+        padding: 0;
+        margin: 0;
+    }
+    li {
```

Don't use child selectors (see lint warning).
```suggestion
    &__checklist-item {
```